src/common/useConfig.ts

```diff
@@ -0,0 +1,9 @@
+import { Config, AuthenticationMethod, Language } from '../api';
+
+export function useConfig(): Config {
```

Comment:
Can you make this an async function so that you won't need to change the consumers later?

Reply:
It's a React hook for usage in components, so for the consumers, it'll be a sync function 👍

Reply:
Still we need to pass some concept of loading that the components can handle imo, or just return null

Reply:
Yup, I'd expect it to return `null` when not initialised.